app/src/pages/inside/settingsPage/generalTab/generalTab.jsx

```diff
@@ -0,0 +1,275 @@
+import React, { Component } from 'react';
+import classNames from 'classnames/bind';
+import PropTypes from 'prop-types';
+import { FormattedMessage } from 'react-intl';
+import { reduxForm, formValueSelector } from 'redux-form';
+import { connect } from 'react-redux';
+import { Input } from '../../../../components/inputs/input';
```

Comment:
import { Input } from 'components/inputs/input'

Fix for all following cases. We use folder aliases on our project.